README.md

```diff
@@ -56,6 +56,7 @@ Bundle of more than 30 new textobjects for Neovim.
 | restOfWindow           | from the cursorline to the last line in the window                                        | \-                                                                                        | \-              |           `gW`           | all                             |
 | lastChange             | Last non-deletion-change, yank, or paste.[^2]                                             | \-                                                                                        | \-              |           `g;`           | all                             |
 | mdlink                 | markdown link like `[title](url)`                                                         | inner is only the link title (between the `[]`)                                           | small           |        `il`, `al`        | markdown, toml                  |
+| mdEmphasis             | markdown text enclosed by `*`, `**`, `***`, `_`, `__`, `___`                              | inner is only the emphasis content title                                                  | small           |        `ie`, `ae`        | markdown, toml                  |
```

Comment:
Probably only copypaste, but I don't that for this textobject, `toml` makes sense?
(for mdLink it makes sense, because toml knows some similar syntax there)